apisix/core/etcd.lua

```diff
@@ -360,4 +361,19 @@ function _M.server_version()
 end
 
 
+function _M.keepalive(id)
+    local etcd_cli, _, err = new()
+    if not etcd_cli then
+        return nil, err
+    end
+
+    local res, err = etcd_cli:keepalive(id)
+    if not res then
+        return nil, err
+    end
+
+    return res, nil
```

Comment:
remove `, nil`

Reply:
If I don't return nil, how can I be sure there is no err?

Reply:
the default value is `nil` in this case

Reply:
I checked the other methods in etcd.lua, push, delete, set, etc. all return nil, and I think I'm correct in returning nil here. Shouldn't the code style of a repository be uniform? Even if the default value is nil, it should be explicitly shown in the code, so that the logic of the code is clearer.

Reply:
`, nil` is useless, so we should remove it and keep the code simpler

Reply:
I checke APISIX repo, other file has the same situation. Example:
- https://github.com/apache/apisix/blob/cb2ba6819f72d75053f8f95aa781975dca2223fd/apisix/balancer/ewma.lua#L93

- https://github.com/apache/apisix/blob/cb2ba6819f72d75053f8f95aa781975dca2223fd/apisix/balancer/ewma.lua#L100
- https://github.com/apache/apisix/blob/cb2ba6819f72d75053f8f95aa781975dca2223fd/apisix/core/schema.lua#L56
- https://github.com/apache/apisix/blob/cb2ba6819f72d75053f8f95aa781975dca2223fd/apisix/core/utils.lua#L137
- https://github.com/apache/apisix/blob/cb2ba6819f72d75053f8f95aa781975dca2223fd/apisix/plugins/basic-auth.lua#L112
- https://github.com/apache/apisix/blob/cb2ba6819f72d75053f8f95aa781975dca2223fd/apisix/plugins/openid-connect.lua#L159

There are more places where there is a similar situation, where nil is returned when err equals nil, do we need to be uniform for those as well?

Reply:
we can create a new issue, fix them later

Reply:
Got it.